src/@gen3/arranger/bin/server.ts

```diff
@@ -33,7 +33,7 @@ app.use(router);
 app.get('/_status', async function(req, res) {
   console.log('Processing /_status');
   const status = await checkHealth();
-  if (!status.isHealthy) {
+  if (!status.isHealthy || !projectStarted) {
```

Comment:
Hey Ted,
Sorry - I didn't even remember that that `healthCheck` module existed.  If you can move the `projectStarted` variable into `healthCheck.ts`, and maybe export `setProjectStarted` and `isProjectStarted` , then:
```
import { setProjectStarted, checkHealth } from '.../healthCheck';
...
setProjectStarted(true)
```
then just have `checkHealth` fail if the project has not been started ...
Something like that ... I think that will be nicer - have all our health related info in `healthCheck.ts`.